src/resources/i18n/microsoft/Messages_fr_FR.xml

```diff
@@ -87,8 +87,9 @@
    <translation id="microsoft_office_license_add_user_confirm" qtlid="264914">L'utilisateur suivant sera ajouté</translation>
    <translation id="microsoft_office_license_add_user_license_none" qtlid="446611">Sélectionnez une licence</translation>
    <translation id="microsoft_office_license_detail_user_add_error" qtlid="264940">Une erreur est survenue lors de l'ajout de l'utilisateur</translation>
-   <translation id="microsoft_office_license_add_user_login_minlength" qtlid="264953">Le nom d'utilisateur doit contenir au minimum 3 caractères</translation>
-   <translation id="microsoft_office_license_add_user_login_maxlength" qtlid="264966">Le nom d'utilisateur est limité à 20 caractères maximum</translation>
+   <translation id="microsoft_office_license_add_user_login_conditions">Attention, le login doit respecter les conditions suivantes : <ul><li>Minimum {0} caractères</li><li>Maximum {1} caractères</li><li>Peut contenir des lettres, des chiffres et les caractères suivants - _ ! # $ % & ' ^ ` { } ~</li><li>Peut contenir le caractère "." s'il est précédé d'un autre caractère autorisé et n'est pas suivi d'un autre "."</li></ul></translation>
```

Comment:
Could you please avoid using HTML tags from here?
It would be better if each `<li>` elements has his own key.